app/javascript/packs/application.jsx

```diff
@@ -3,33 +3,35 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
-require("@rails/ujs").start();
-require("turbolinks").start();
-require("@rails/activestorage").start();
-require("channels");
+require('@rails/ujs').start()
+require('turbolinks').start()
+require('@rails/activestorage').start()
+require('channels')
 
-import "css/application.css";
+import 'css/application.css'
 
-import React from "react";
-import { initReact } from "./react_bootloader.jsx";
-import { ExampleIterationsSummaryTable } from "../components/example/iterations_summary_table.jsx";
-import { MaintainingIterationsSummaryTable } from "../components/maintaining/iterations_summary_table.jsx";
+import React from 'react'
+import { initReact } from './react_bootloader.jsx'
+import { ExampleIterationsSummaryTable } from '../components/example/iterations_summary_table.jsx'
+import { MaintainingIterationsSummaryTable } from '../components/maintaining/iterations_summary_table.jsx'
+import { MentorConversationsList } from '../components/mentoring/mentor_conversations_list.jsx'
```

Comment:
FYI, it's very unusual to see snake case for JavaScript files.

Reply:
I agree with renaming too!